src/index.scss

```diff
@@ -25,6 +25,7 @@
     --color--text: hsl(240, 10%, 10%);
     --color--background: var(--color--reddit-gray);
     --color--background--textarea: white;
+    --color--background--textarea: #F6F7F8;
```

Comment:
With CSS, newest declaration takes precedence. If you want to change the background color of the textarea, just replace the "white" on line 27.

Reply:
Sorry for the bad example with `--color--text`, but a color like this should be put in a variable in the top-level :root block above. Then, colors like that should be referred to in the prefers-color-scheme blocks, like how `--color-background` is.